cmake/Reprojucer.cmake

```diff
@@ -3609,6 +3609,9 @@ function(_FRUT_set_compiler_and_linker_settings target)
       target_compile_options(${target} PRIVATE
         "-mmacosx-version-min=${osx_deployment_target}"
       )
+      set_property(TARGET ${target} APPEND_STRING PROPERTY LINK_FLAGS
```

Comment:
Good job on figuring out that `LINK_FLAGS` is a string, not a list. This means we need to fix other places where we are appending to `LINK_FLAGS` as if it were a list. But I can do that on another PR.